README.md

```diff
@@ -99,13 +111,27 @@
 
 ![Screenshot](img/spellBeeInstruct.PNG)
 
-### Running Tests
+### Testing Suite
 
-    Command to run pytest (Be sure to be in environment before using)
-        Run All:
-            - (pytest UnitTests/)
-        Run Specific:
-            - (pytest UnitTests/TestFileName)
+    Using the Testing Suite
+
+    Running a complete Test
+        - Use the command line command ( (Test) or (Test all))
+            -To add coverage use ( (Test cov) or (Test all cov))
```

Comment:
Paren spacing looks bad ( (xyz))

Reply:
Changed style to better emphasize commands